### PR TITLE
sci-physics/pythia: Install lhapdf plugin lib

### PR DESCRIPTION
**Pull request by APN-Pucky**
Signed-off-by: Alexander Puck Neuwirth <alexander@neuwirth-informatik.de>

### CODE REVIEW

**Comment by APN-Pucky** [comment]
@gentoo/sci-physics ?

File: sci-physics/pythia/pythia-8.2.45-r1.ebuild [hunk 1]
```diff
@@ -159,6 +159,7 @@ src_install() {
 	dobin bin/pythia8-config
 	doheader -r include/*
 	dolib.so lib/libpythia8.so
+	use lhapdf && dolib.so lib/libpythia8lhapdf6.so
```
Comment (AndrewAmmerlaan) [inline sci-physics/pythia/pythia-8.2.45-r1.ebuild:162]: This is going to require a revbump in order to propagate this change to all users.

**Comment by APN-Pucky** [comment]
Thanks, I was not sure if I should directly add the revbumped version, since the differences aren't so nicely visible then. 
I'll bump it soon.

**Comment by AndrewAmmerlaan** [comment]
Just to double check, this change is only required for version 8.2.45 and not for the latest version 8.3.07?

**Comment by APN-Pucky** [comment]
Thanks for checking. It is also in 8.3.* and I fixed the URL.

**Comment by AndrewAmmerlaan** [comment]
Thanks :+1:

**Pull request closed without merging**